File: frontend/ksweb-app/src/app/pages/status/status.models.ts
export interface StatusListItem {
  statusId: number;
  statusName: string;
  isPending: boolean;
  statusStopClock: boolean;
  statusDescription: string;
  isDeleted: boolean;
  internalName: string;
}

export interface StatusForm {
  statusName: string;
  isPending: boolean;
  statusStopClock: boolean;
  statusDescription: string;
  internalName: string;
}

export interface StatusFiltro {
  page: number;
  pageSize: number;
  termo: string;
}

File: backend/KSWeb.Api/Models/StatusModels.cs
namespace KSWeb.Api.Models;

public sealed class StatusListItem
{
    public long StatusId { get; set; }
    public string StatusName { get; set; } = string.Empty;
    public bool IsPending { get; set; }
    public bool StatusStopClock { get; set; }
    public string StatusDescription { get; set; } = string.Empty;
    public bool IsDeleted { get; set; }
    public string InternalName { get; set; } = string.Empty;
}